packages/serve-cli/src/run.ts

```diff
@@ -80,7 +85,20 @@ let program = new Command()
       }),
   )
   .option('--masked-errors', 'mask unexpected errors in responses')
-  .option('--subgraph <path>', 'path to the subgraph schema');
+  .addOption(
+    new Option('--subgraph <path>', 'path to the subgraph schema').conflicts([
+      'supergraph',
+      'proxy',
+    ]),
+  )
+  .addOption(
+    new Option('--proxy <endpoint>', 'proxy all requests to this endpoint')
+      .env('PROXY_ENDPOINT')
+      .argParser(e => ({
+        endpoint: new URL(e).toString(),
+      }))
+      .conflicts(['supergraph', 'subgraph']),
+  );
```

Comment:
Can we also add these options for the CDN access key and the CDN supergraph/SDL endpoint?

Reply:
It is part of another PR;
https://github.com/ardatan/graphql-mesh/pull/7469

Reply:
okay then 🟢 from me, let's wait for @enisdenjo to review and then integrate it into https://github.com/ardatan/graphql-mesh/pull/7469!

Reply:
I'll merge this then @enisdenjo can continue in his PR.